### PR TITLE
compatibility V1.5.11

### PR DESCRIPTION
**Pull request by HiddeLekanne**

### CODE REVIEW

File: events/opium_wars_events.txt [hunk 1]
```diff
@@ -367,9 +382,12 @@ opium_wars.3 = {
 		}
 		add_modifier = {
 			name = china_welfare
-			months = normal_modifier_time
+			months = very_long_modifier_time
```
Comment (HiddeLekanne) [inline events/opium_wars_events.txt:386]: Whether this is a good change or not....

File: events/opium_wars_events.txt [hunk 2]
```diff
@@ -158,9 +158,16 @@ opium_wars.1 = {
...
+			years = 40 # very_long_modifier_time
 			is_decaying = yes
 		}
+		add_radicals = {
+			value = medium_radicals
+		}		
+		if = {
+			limit = { has_journal_entry = je_warlord_china }
+			chi_remove_1_fragile_unity = yes
+		}
```
Comment (Danarca) [inline events/opium_wars_events.txt]: We need to implement this regarding the changed approach to fragile unity...

Comment (HiddeLekanne) [reply events/opium_wars_events.txt]: Do we have a problem with fragile unity and warlords?

Comment (Danarca) [reply events/opium_wars_events.txt]: Fragile unity has gone from a scale from 0 - 5 to 0 - 100. And it uses a scripted value to calculate how much it has.
So these lines would do nothing.

File: events/opium_wars_events.txt (hunk 1 above)
Comment (Emifas) [reply events/opium_wars_events.txt:386]: Should be fine, China isn't really designed to win the Opium War in our mod anyway

File: events/opium_wars_events.txt (hunk 2 above)
Comment (Emifas) [reply events/opium_wars_events.txt]: The script value needs to be updated

Comment (HiddeLekanne) [reply events/opium_wars_events.txt]: I made it a todo

**Pull request merged**